From 0e82d5256abc7757d17bea77da01570d2b21d7c1 Mon Sep 17 00:00:00 2001
From: Daniel Kurtz <djkurtz@chromium.org>
Date: Wed, 27 Jan 2016 19:49:58 +0800
Subject: [PATCH libdrm 1/7] CHROMIUM: Add Mediatek proprietary format

Add DRM_FORMAT_MT21 for Mediatek Properity video block format.

Old-BUG=chrome-os-partner:38394
Old-BUG=b:26575005
Old-TEST=oak: Boot to UI.

Signed-off-by: Chiawen Lee <chiawen.lee@mediatek.com>
Reviewed-on: https://chromium-review.googlesource.com/324183
Commit-Ready: Nicolas Boichat <drinkcat@chromium.org>
Tested-by: Nicolas Boichat <drinkcat@chromium.org>
Reviewed-by: Daniel Kurtz <djkurtz@chromium.org>
(rebased from chromeos-2.4.89)

BUG=b:112720782
Signed-off-by: Kristian H. Kristensen <hoegsberg@chromium.org>

Change-Id: I3151bbd41c1473deb73d88ba375a6a9e218c9423
---
 include/drm/drm_fourcc.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/drm/drm_fourcc.h b/include/drm/drm_fourcc.h
index e04613d3..d4e92d1f 100644
--- a/include/drm/drm_fourcc.h
+++ b/include/drm/drm_fourcc.h
@@ -141,6 +141,8 @@ extern "C" {
 #define DRM_FORMAT_NV24		fourcc_code('N', 'V', '2', '4') /* non-subsampled Cr:Cb plane */
 #define DRM_FORMAT_NV42		fourcc_code('N', 'V', '4', '2') /* non-subsampled Cb:Cr plane */
 
+#define DRM_FORMAT_MT21		fourcc_code('M', 'T', '2', '1') /* Mediatek Block Mode */
+
 /*
  * 3 plane YCbCr
  * index 0: Y plane, [7:0] Y
-- 
2.16.4

